presto-main/src/main/java/com/facebook/presto/sql/analyzer/QueryExplainer.java

```diff
@@ -168,20 +168,23 @@ public String getJsonPlan(Session session, Statement statement, Type planType, L
     }
 
     public Plan getLogicalPlan(Session session, Statement statement, List<Expression> parameters, WarningCollector warningCollector)
+    {
+        return getLogicalPlan(session, statement, parameters, new PlanNodeIdAllocator(), warningCollector);
+    }
+
+    public Plan getLogicalPlan(Session session, Statement statement, List<Expression> parameters, PlanNodeIdAllocator idAllocator, WarningCollector warningCollector)
```

Comment:
It might be better to put `idAllocator` as the last parameter since we have a default value from overloaded `getLogicalPlan`.